src/infra/pg-secrets.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pg-secrets
+type: Opaque
+data:
+  POSTGRES_USERNAME: cG9zdGdyZXM=
+  POSTGRES_PASSWORD: cG9zdGdyZXM=
+  POSTGRES_URL: amRiYzpwb3N0Z3Jlc3FsOi8vcG9zdGdyZXM6NTQzMi9wb3N0Z3Jlcz9jdXJyZW50U2NoZW1hPWV2ZW50c19zY2hlbWE=
```

Comment:
Things like database url I'm not recommend to encrypt. Traceability of the project will be better, if you will use the configmap and env only the host name.
Like ```PG_HOST```, while url: ```jdbc:postgresql://${PG_HOST}:5432/postgres```

Remember it's all about maintainability and traceability of the project!